harness/features/initialize.local.test.ts

```diff
@@ -88,15 +88,13 @@ describe('Initialize Tests - Local', () => {
         const testClient = new LocalTestClient(sdkName)
         const configRequestUrl = testClient.getValidConfigPath()
 
-        scope.get(configRequestUrl).times(2).reply(500)
+        // a 500 is a recoverable error so it will retry the connection and potentially make more requests than expected.
```

Comment:
why would it make more requests than the polling interval? or is the interval timer only started at successful call?

Reply:
Correct - the interval timer starts at the successful call.

Reply:
for all sdks? if that was the case would we have caught this or saw inconsistencies earlier?

Reply:
not fully confident that this wouldnt break other sdks.. but i guess if the tests pass then we can merge it..